modules/openapi-generator/src/main/resources/dart/libraries/dio/serialization/built_value/simple_type.mustache

```diff
@@ -0,0 +1 @@
+{{#isEnum}}{{{enumName}}}{{/isEnum}}{{^isEnum}}{{{datatypeWithEnum}}}{{^datatypeWithEnum}}{{{dataType}}}{{/datatypeWithEnum}}{{/isEnum}}
```

Comment:
This does not look specific to built_value. We should try to keep generic handling outside of the `built_value` folder so that it can be reused for the upcoming `json_serializable` support.